cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -428,6 +441,7 @@ defaultPartialNodeConfiguration =
     , pncLogMetrics = mempty
     , pncTraceConfig = mempty
     , pncMaybeMempoolCapacityOverride = mempty
+    , pncLedgerDBBackend = Last . Just $ LMDB
```

Comment:
Is `LMDB` the default?

Reply:
I think so (for normal users it should be LMDB), we can always revert it back